src/message.ts

```diff
@@ -613,6 +633,81 @@ export class Message implements Sayable {
   // }
 
   /**
+   *
+   * @param sendTo Room or Contact
+   * @param option Whether to display the message source, and the custom source information format
+   */
+  public forward(room: Room, option?: ForwardOption): Promise<boolean>
+  public forward(contact: Contact, option?: ForwardOption): Promise<boolean>
+  public forward(sendTo: Room|Contact, option?: ForwardOption): Promise<boolean> {
```

Comment:
I think the `forward` methods should be moved from the class `Message` to `MediaMessage`.